proto/admin.proto

```
@@ -0,0 +1,149 @@
+syntax = "proto3";
```

Buf (1.67.0)

[error] Files must have a package defined.

(PACKAGE_DEFINED)